source/winAPI/sessionTracking.py

```diff
@@ -157,9 +157,9 @@ def registerSessionNotification(handle: HWND) -> bool:
 	return registrationSuccess
 
 
-def unregisterSessionNotification(handle: HWND) -> None:
+def unregister(handle: HWND) -> None:

```

Comment:
Not sure why I didn't notice this before, but this should probably be `deregister`. See: https://english.stackexchange.com/a/40095